code/modules/admin/fun_verbs.dm

```diff
@@ -1000,27 +1000,45 @@
 	message_admins("[ADMIN_TPMONTY(IF)] started being imaginary friend of [ADMIN_TPMONTY(L)].")
 
 
-	
 /datum/admins/proc/force_dropship()
 	set category = "Fun"
 	set name = "Force Dropship"
 
 	if(!check_rights(R_FUN))
 		return
 
-	if(!length(SSshuttle.dropships))
+	if(!length(SSshuttle.dropships) && !SSshuttle.canterbury)
 		return
 
-	var/obj/docking_port/mobile/marine_dropship/D = SSshuttle.dropships[1]
+	var/list/available_shuttles = list()
+	for(var/i in SSshuttle.mobile)
+		var/obj/docking_port/mobile/M = i
+		available_shuttles.Add(M.id)
 
-	if(!istype(D))
+	var/shuttle_id = input(usr, "Which shuttle do you want to move?", "Force Dropship") as null|anything in available_shuttles
+	if(!shuttle_id)
```

Comment:
Shuttle ID doesn't have to be invalid if they just press cancel, this message is misleading